Cargo.toml

```diff
@@ -69,19 +69,19 @@ miden-assembly = { git = "https://github.com/0xPolygonMiden/miden-vm", rev = "0a
 miden-core = { git = "https://github.com/0xPolygonMiden/miden-vm", rev = "0ac54125d5bd43160566736b2be3f5818fd84f69" }
 miden-processor = { git = "https://github.com/0xPolygonMiden/miden-vm", rev = "0ac54125d5bd43160566736b2be3f5818fd84f69" }
 miden-stdlib = { git = "https://github.com/0xPolygonMiden/miden-vm", rev = "0ac54125d5bd43160566736b2be3f5818fd84f69" }
-midenc-codegen-masm = { path = "codegen/masm" }
+midenc-codegen-masm = {version = "0.0.0", path = "codegen/masm" }
```

Comment:
Is this actually necessary? I'd expect each of these to be resolved to whatever the `version` is in that sub-crate's `Cargo.toml`. By specifying a version here, that feels super brittle, since you'd have to change the version in both places when a change to that crate's version happens.

Reply:
Good point! Removed.

Reply:
I was thinking that `release-plz` requires it, but it does not.